t/db_dependent/api/v1/holdings.t

```diff
@@ -45,9 +46,48 @@ my $t = Test::Mojo->new('Koha::REST::V1');
 my $tx;
 
 subtest "list() tests" => sub {
-    plan skip_all => 'not implemented';
+    plan tests => 12;
 
     $schema->storage->txn_begin;
+
+    my ($patron, $session_patron) = create_user_and_session( { authorized => 0 } );
+
+    Koha::Holdings->search->delete;
+
+    $tx = $t->ua->build_tx( GET => '/api/v1/holdings' );
+    $tx->req->cookies( { name => 'CGISESSID', value => $session_patron } );
+    $t->request_ok( $tx )->status_is( 200 )
+        ->json_is( [] );
+
+    my $biblionumber = create_biblio( 'test' );
+
+    # test with 1 holding
+    my $holding = create_holding( $biblionumber );
+    my $holdings_full = Koha::Biblios->find( $biblionumber )->holdings_full();
+    $tx = $t->ua->build_tx( GET => '/api/v1/holdings' );
+    $tx->req->cookies( { name => 'CGISESSID', value => $session_patron } );
```

Comment:
There is a gigaton of repetition all over the place with this specific line of code being repeated ad-infinitum in all REST API tests.
```$tx->req->cookies( { name => 'CGISESSID', value => $session_patron } );```
The old REST API test suite puts the authentication token to the Mojo::UserAgent cookie jar, so you can forget about authentication for every request.

Reply:
Repetition sucks but implementing another approach is out of scope of this Bug. However, Koha Community has began using Basic Access authenticaiton in API tests, so that specifying the cookie eventually becomes obsolete.